src/includes/performance/traces-sampler-as-filter/java.mdx

```diff
@@ -0,0 +1,37 @@
+```java
+import io.sentry.CustomSamplingContext;
```

Comment:
please add for Android too https://sentry-docs-git-fork-maciejwalkowiak-java-sampling-filter.sentry.dev/platforms/android/configuration/filtering/#using-sampling-to-filter-transaction-events
otherwise it inherits from Java but the static class changes